scadalts-ui/src/views/AlarmTabs.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+	<div class="alarms">
+		<v-card>
+			<v-tabs
+			v-model="tab"
+			@change="changeTab"
+			>
+				<v-tab
+					v-for="item in items"
+					:key="item.tab"
+					:class="{'blink':(item.tab === 'plc' && newAlarms)}"
+				>
+					{{ item.content }}
+				</v-tab>
+			</v-tabs>
+		</v-card>				
+        <router-view></router-view>
+
+	</div>
+</template>
+
+<script>
+import Components from '@min-gb/vuejs-components';
+
+export default {
+	name: 'alarmTabs',
+	components: {
+		...Components,
+	},
+	mounted() {
+		this.tab = 0
+		this.changeTab(this.tab)
+		const interval = setInterval(() => {
+			this.getAlarms();
+		}, 1000);
```

Comment:
If you setInterval on "mounted" hook you **must** clear it using "beforeDestroy" hook. `clearInterval(...)`. If you do not do that this interval will be triggered even if you are on the other tab and for example you are adding a user.